src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -319,28 +362,37 @@ const BlastSubmissionName = () => {
   );
 };
 
-// to be replaced with a simple select component
-const buildSelect = (setting: {
+type BlastSelectProps = {
   options: Option[];
   label: string;
+  description?: string;
   selectedOption: string;
   onChange: (value: string) => void;
-}) => {
+};
+
+const BlastSelect = (setting: BlastSelectProps) => {
   const onChange = (e: FormEvent<HTMLSelectElement>) => {
     const value = e.currentTarget.value;
     setting.onChange(value);
   };
 
+  const [hoverRef, isHovered] = useHover<HTMLSpanElement>();
+
   return (
     <div className={styles.select}>
       <label>
-        <span>{setting.label}</span>
+        <span ref={hoverRef}>{setting.label}</span>
         <SimpleSelect
           value={setting.selectedOption}
           onInput={onChange}
           options={setting.options}
         />
       </label>
+      {isHovered && setting.description && (
+        <Tooltip anchor={hoverRef.current} autoAdjust={true}>
+          {setting.description}
```

Comment:
I don't think simple `<br>` would even work here. It would be printed as plain text inside of the tooltip. To make those `<br>''s work, you would have to `dangerousySetInnerHtml` on something. But it would be simpler without them.

We'd have to wrap the description on line 393 in a div. If you absolutely must split the lines as in the XD, then ask Andres to use a line break character there; and use a `white-space: pre-wrap` css rule on that wrapper div.